scripts/devnet-curio/curio.dockerfile

```diff
@@ -0,0 +1,103 @@
+#####################################
+ARG LOTUS_IMAGE
+# hadolint ignore=DL3006
+FROM ${LOTUS_IMAGE} as lotus-test
+FROM golang:1.22.3-bullseye AS curio-builder
```

Comment:
```suggestion
FROM golang:1.22-bookworm AS curio-builder
```

Reply:
The curio team uses `golang:1.22.3-bullseye`, so I hesitate to deviate. If `bookworm` is better, maybe we can upstream the change.